doc/source/aggregation_with_dissolve.rst

```diff
@@ -0,0 +1,51 @@
+.. ipython:: python
+   :suppress:
+
+   import geopandas as gpd
+
+
+Aggregation with `dissolve`
+===========================
+
+It is often the case that we find ourselves working with spatial data that is more granular than we need. For example, we might have data on sub-national units, but we're actually interested in studying patterns at the level of countries.
+
+In a non-spatial setting, we aggregate our data using the `groupby` function. But when working with spatial data, we need a special tool that can also aggregate geometric features. In the *geopandas* library, that functionality is provided by the `dissolve` function.
+
+`dissolve` can be thought of as doing three things: (a) it dissolves all the geometries within a given group together into a single geometric feature (using the `unary_union` method), and (b) it aggregates all the rows of data in a group using `groupby.aggregate()`, and (c) it combines those two results.
+
+`dissolve` Example
+~~~~~~~~~~~~~~~~~~
+
+Suppose we are interested in studying continents, but we only have country-level data like the country dataset included in *geopandas*. We can easily convert this to a continent-level dataset.
+
+
+First, let's look at the most simple case where we just want continent shapes and names. By default, `dissolve` will pass `'first'` to `groupby.aggregate`, which can be very useful for getting the names of an aggregated shape.
```

Comment:
yeah -- problem with example data -- no extra string variables. It comes up a lot in practice when one has variables that are already at the level to which one is aggregating. For example, if one is aggregating states to countries and there's a column for country leader's name, or country code, etc. But wasn't sure how to deal with given existing example data. I'll just drop it.